1-exercises/F-for-of-loop/exercise.js

```diff
@@ -11,6 +11,14 @@ let tubeStations = [
     "Tottenham Court Road"
 ];
 
+for(let stations of tubeStations){
+    console.log(stations);
+}
 
 // TODO Use a for-of loop to capitalise and output each letter in the string seperately.
 let str = "codeyourfuture";
+
+for(let letter of str){
+    let capitaliseLetter = letter.toUpperCase();
+    console.log(capitaliseLetter);
+}
```

Comment:
Good you have clearly understood how to use a for loop on an array.

Reply:
Thank you